src/hbbft.erl

```diff
@@ -355,6 +355,8 @@ handle_msg(Data = #hbbft_data{round=R}, J, {dec, R, I, Share}) ->
                                                                     failed_decrypt=[I|Data#hbbft_data.failed_decrypt]});
                                 Decrypted ->
                                     case decode_list(Decrypted, []) of
+                                        [_ | Transactions] when length(Transactions) > Data#hbbft_data.batch_size ->
+                                            ignore;
```

Comment:
Should we ignore or truncate here? @evanmcc thoughts?

Reply:
that's a good question!  I'm not actually sure what would be best.  my feeling is that either is fine?  dropping makes it a lot easier to test, and maybe easier to make deterministic?

Reply:
@Vagabond If we truncate, how do we know which to drop and which to keep? If I'm following the chain of calls correctly - from here it would go into `{result, {transactions, ...` which the miner in turn accepts as final.

Reply:
you could just truncate to the first *batch size* elements, the commitment is already agreed to so everyone will have the same batch, in the same order.

Dropping is probably fine though.